dockerfiles/e2e-saas/docker-entrypoint.sh

```diff
@@ -143,7 +143,6 @@ export DISPLAY=:1.0
 
 echo "Applying patches..."
 # currently in /tmp/rh-che folder
-patch -u ../e2e/pageobjects/dashboard/Dashboard.ts -i ../test_patches/Dashboard.patch
```

Comment:
@Katka92 so, in 7.11 e2e tests are not dependent on the UD tabs?

Reply:
They are, but only on Workspaces, Stacks and Getting started.